dist/images/ovnkube.sh

```diff
@@ -165,21 +201,21 @@ wait_for_northd () {
       ovn-nbctl show > /dev/null 2>&1
       if [[ $? != 0 ]] ; then
         (( retries += 1 ))
-        if [[ "${retries}" -gt 40 ]]; then
+        if [[ "${retries}" -gt 80 ]]; then
           echo "error: ovn-northd did not come up, exiting"
```

Comment:
Any way we can abstract all the "wait for XXX" into a single function that gets passed a command to call or something?

Reply:
Working on this